src/openforms/formio/dynamic_config/referentielijsten.py

```diff
@@ -6,14 +6,14 @@
 from zgw_consumers.models import Service
 
 from openforms.contrib.referentielijsten.client import ReferentielijstenClient
+from openforms.contrib.referentielijsten.helpers import check_expired_or_near_expiry
 from openforms.logging import logevent
 from openforms.submissions.models import Submission
-
-from ..typing import Component
+from openforms.typing import JSONObject
 
 
 def fetch_options_from_referentielijsten(
-    component: Component, submission: Submission
+    component: JSONObject, submission: Submission
```

Comment:
that's a wrong type - it's supposed to represent a Formio component

Reply:
This was failing in pyright that's why I changed it. (Basically all the logs have `component: JSONObject` and then I would have to change them all-don't know if it is what we want)

Reply:
Let's just ignore it for now with a `# pyright: ignore[<errorType>]` comment and add a TODO to clean up the ignore